### ISSUE TITLE
Thank you for this initiative!

### ISSUE THREAD

**Issue by mindreframer** [nalgeon/sqlpkg-cli#1]
Sorry, could not find a better channel to express my gratitude! This is a great project that fills a much needed niche to handle SQLITE extensions in a predictable and automatic way! 

I'm using a MacBook M1, and quite some extensions are not available for `mac arm` architecture. Is there anything the community could help with to fill this void? I would really love to be able to install all of those current 58 extensions on a MacBook AIR M1 just by using sqlpkg. What would be the best way to contribute?

Cheers and have a great week start! 
Roman

**Comment by nalgeon** [comment]
You are very welcome! I've just updated all the extensions [I maintain](https://github.com/nalgeon/sqlean/issues/27) to support macOS Apple processors. So you can install them with the `sqlpkg`.

As for extensions maintained by others (it's mostly @asg017, I think) — you'll have to ask them to add such support to their extensions. After that, I'll update the package specs so that `sqlpkg` can handle them as well.

**Comment by mindreframer** [comment]
I see. When I tried to install asg017/vss, it was missing the vector extension, that has no package for ARM. 

Then I took the release directly from https://github.com/asg017/sqlite-vss/releases. (https://github.com/asg017/sqlite-ecosystem#sqlite-vss) - https://github.com/asg017/sqlite-vss/releases/download/v0.1.1-alpha.20/sqlite-vss-v0.1.1-alpha.20-loadable-macos-aarch64.tar.gz. 
And it has the vector extensions zipped into the archive: 

```bash 
./sqlite-vss-v0.1.1-alpha.20-loadable-macos-aarch64
./sqlite-vss-v0.1.1-alpha.20-loadable-macos-aarch64/vector0.dylib
./sqlite-vss-v0.1.1-alpha.20-loadable-macos-aarch64/vss0.dylib
```
I've tried it, it works. I only needed to mark it a as allowed binary with `xattr -d com.apple.quarantine *`. 

Granted this is an unusual case and there are not standards right now... This is why it's so great to have a single place with all the extensions, so it's possible to come up with some standards for compilation / distribution. 


But yeah, I'll bug @asg017 if I need one of his extensions to work on ARM.

Thanks and have a great week!
Roman
